Dockerfile

```diff
@@ -0,0 +1,16 @@
+# -- Stage 1 -- #
+# Compile the app.
+FROM golang:1.16-alpine as builder
+WORKDIR /app
+COPY go.mod go.sum ./
+RUN go mod download
+COPY . .
```

Comment:
might be better to not copy environment variables and secrets into the docker image, maybe you can just copy the folders and files individually so that env files and scripts are not copied?